README.md

```diff
@@ -284,6 +284,7 @@ Here's a (non-exhaustive) list of companies that use `rules_scala` in production
 * [Meetup](https://meetup.com/)
 * [Spotify](https://www.spotify.com/)
 * [Stripe](https://stripe.com/)
+* [Airbnb](https://airbnb.com/)
```

Comment:
@thomasbao12 could you please move the entry to the top, so it stays sorted alphabetically?